auth0/dto.go

```diff
@@ -0,0 +1,94 @@
+package auth0
+
+import "time"
+
+// AuthToken Struct
+type AuthToken struct {
+	Name      string    `json:"name"`
+	Token     string    `json:"token"`
+	CreatedAt time.Time `json:"created_at"`
+}
+
+// Resp struct
+type Resp struct {
+	AccessToken string `json:"access_token"`
+	Scope       string `json:"scope"`
+	ExpiresIn   int    `json:"expires_in"`
+	TokenType   string `json:"token_type"`
+}
+
+// ESTokenSchema for AuthToken
+type ESTokenSchema struct {
```

Comment:
Same here with tokens - what do we need token for in V2?

Reply:
Yes, you are right, it's just temp will be removed

Reply:
let me remove it now

Reply:
Thanks! basically all auth0/profile related stuff.
I remember on standup - we agreed that those new DSes will output identities and that's it, so someting like SH DB `identities` table - but not `profiles` table.
Just identity data like: `(UUID, source, name, username, email)`.